karapace/config.py

```diff
@@ -112,13 +112,13 @@ def set_config_defaults(config: Config) -> Config:
 
 
 def write_config(config_path: Path, custom_values: Config) -> None:
-    config_path.write_text(json.dumps(custom_values))
+    config_path.write_text(ujson.dumps(custom_values))
 
 
 def read_config(config_handler: IO) -> Config:
     try:
-        config = json.load(config_handler)
-    except json.JSONDecodeError as ex:
+        config = ujson.load(config_handler)
+    except ValueError as ex:
```

Comment:
Error class will get corrected in the next release of ujson.
https://github.com/ultrajson/ultrajson/issues/497